src/sql/src/plan/statement/ddl.rs

```diff
@@ -1090,6 +1092,7 @@ fn get_encoding_inner<T: sql_parser::ast::AstInfo>(
                 DataEncoding::Protobuf(ProtobufEncoding {
                     descriptors,
                     message_name: NormalizedProtobufMessageName::new(message_name.to_owned()),
+                    confluent_wire_format: false,
```

Comment:
@benesch @quodlibetor this is probably the part of the code I'm most unfamiliar with, so making sure these are configured correctly is definitely something to make sure I did right!

Reply:
This LGTM, I added a different comment expanding on testing the unhappy path.